### PR TITLE
Improve version coverage for the rosa/rosa hcp clusters

### PR DESCRIPTION
**Pull request by yasun1**
[OCPQE-18519](https://issues.redhat.com//browse/OCPQE-18519)

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse  periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-aws-rosa-sts-advanced-f28 periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-rosa-hcp-advanced-full-f14

**Comment by yasun1** [comment]
@liangxia Please help review. Thanks. 

cc @heliubj18 Just let you know, I also add more nightly jobs for the HCP clusters on 4.15/4.16

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -1284,6 +1284,57 @@ tests:
...
+    test:
+    - chain: openshift-e2e-test-hypershift-qe
+    workflow: rosa-aws-sts-hypershift-advanced
+- as: aws-rosa-sts-hcp-private-link-full-f14
+  cron: 31 22 9,24 * *
+  steps:
+    cluster_profile: aws-sd-qe
+    env:
+      CHANNEL_GROUP: nightly
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @rosa and @hypershift-hosted and @critical
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1331]: The test name indicates it's a `full` test, but the test filter indicates it's `critical` test. mismatch ?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-stable.yaml [hunk 2]
```diff
@@ -197,6 +197,16 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-usertags-custom-security-groups
+- as: aws-ipi-usertags-fips-f14
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-stable.yaml:200]: Is it a rosa job ?

Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-stable.yaml]: No. Let me remove it

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml (hunk 1 above)
Comment (yasun1) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml]: Forget removing the label. Thanks.

**Pull request merged**